docs/how-tos/adding-events-to-event-bus.rst

```diff
@@ -1,42 +0,0 @@
-Using the Open edX Event bus
```

Comment:
This was moved to this file: https://github.com/openedx/openedx-events/blob/3954196516163fa6f31561885e251e63c6c6e50b/docs/how-tos/using-the-event-bus.rst

Reply:
Maybe we should add a redirect here?